api/Document.json

```diff
@@ -11645,9 +11677,15 @@
                 "version_added": "1.0"
               }
             ],
-            "webview_android": {
-              "version_added": "4.4.3"
-            }
+            "webview_android": [
+              {
+                "version_added": "4.4.3"
```

Comment:
This doesn't quiet square with the data for Chrome. The unprefixed variant was added after the prefixed one, but the order is reversed here, presumably because we're trusting the original "4.4.3" here.

If we knew when prefixed and unprefixed variants were added in Safari we could probably update this data with confidence. My best guess from current data is that we should set both to "4.4.3".

Reply:
Well, the `≤37` is to say that it's in some version between 1 and 37, we just don't know when exactly.  Honestly, though, it would be more accurate if the version were `≤4.4`, since we can determine versions of 4.4 and 4.4.3.

Reply:
Ah, I see. This is fine then, could possibly be resolved later when fixing ≤37 data.